drivers/spi/spi_ll_stm32.c

```diff
@@ -711,7 +712,7 @@ static int transceive_dma(const struct device *dev,
 
 	ret = spi_stm32_configure(dev, config);
 	if (ret != 0) {
-		return ret;
+		goto end;
```

Comment:
should also take this opportunity to align code ?
`	if (ret != 0) {`
vs
`	if (ret) {`